pkg/vault/contracts/ProtocolFeeController.sol

```diff
@@ -134,9 +134,16 @@ contract ProtocolFeeController is
         _;
     }
 
+    modifier withValidPoolCreatorFee(uint256 newPoolCreatorFeePercentage) {
```

Comment:
The check was already being done, but the others have a modifier, so adding this here makes it more consistent.